src/vm/inlinetracking.cpp

```diff
@@ -123,9 +125,72 @@ InlineTrackingEntry & InlineTrackingEntry::operator = (const InlineTrackingEntry
     return *this;
 }
 
+void InlineTrackingEntry::Add(PTR_MethodDesc inliner)
+{
+    STANDARD_VM_CONTRACT;
+
+    MethodInModule method(inliner->GetModule(), inliner->GetMemberDef());
 
-#ifndef DACCESS_COMPILE
-COUNT_T PersistentInlineTrackingMap::GetInliners(PTR_Module inlineeOwnerMod, mdMethodDef inlineeTkn, COUNT_T inlinersSize, MethodInModule inliners[], BOOL *incompleteData)
+    // Going through last 10 inliners to check if a given inliner has recently been registered.
```

Comment:
This number didn't actually come from me, I was simply rearranging the file a bit so that methods on different types wouldn't be intermingled. The original author was Eugene and I think he came up with this number and several others empirically based on some ad-hoc experiements/guessing reasonable values.